### DISCUSSION TITLE
Novità sceme

### DISCUSSION THREAD

**Discussion by pdonadeo** [pdonadeo/e-uscito-joypad#17] [General]
Non so se siete andati oggi su https://www.euscitojoypad.it/

Ho cambiato leggermente il testo, passati i 14 giorni... 😂 

@zampale

**Comment by agofa** [comment]
Bè mi sembra il minimo ....

**Reply by zampale** [reply]
Severo ma giusto.
Comunque registriamo giovedì, live su Twitch. Cosi, info in anteprima solo per voi.

**Reply by DGuidi** [reply]
se si trovasse il modo di abolire festival di sanremo ed eurofestival avremmo puntate più spesso

**Reply by pdonadeo** [reply]
@DGuidi e poi come lo facciamo il sito euscitoilprevedibilepodcast.it con relativo motore di ricerca dei brani? 😂

**Reply by agofa** [reply]
> Severo ma giusto. Comunque registriamo giovedì, live su Twitch. Cosi, info in anteprima solo per voi.

Sì, OK, ma a che ora? (vado vado, non serve spingere) ;-)
